packages/contracts-bedrock/contracts/libraries/BridgeConstants.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+library BridgeConstants {
+    uint32 constant ETH_TX = 0;
+    uint32 constant BIT_TX = 1;
+    uint32 constant ERC20_TX = 2;
```

Comment:
I suggest to unify the naming style. `ETH_TX` -> `ETH_BRIDGE_TX`, `ETH_TX` -> `ETH_BRIDGE_TX`, `ERC20_TX` -> `ERC20_BRIDGE_TX`

Reply:
fixed